prisma/seed.ts

```diff
@@ -117,7 +117,7 @@ async function seed() {
       privateKey:
         '-----BEGIN CERTIFICATE-----ApfFCv0O65TYkp5jEWSlU8PhKYD43nXA=-----END CERTIFICATE-----',
       validFrom: new Date(),
-      validTo: certExpDate,
+      validTo: new Date(),
```

Comment:
Could you please remind me what is the reason for this change?
It seems inclrrect to set the validFrom and validTo both to the same date

Reply:
Typo. It should should just validTo that should be set to new Date.

Reply:
What I mean is
```
validFrom: new Date(),
validTo: new Date(),
```
will create a (mock) certificate with a 0 valid time duration. We should either move validFrom to the past, or validTo to the future (as it was originally)

I don't see the reason we are doing this change... am I missing something?

Reply:
I am fetching certificates based on expiration. I wanted an entry that will always expire in less than a month so we can send a notification. That's why I set validTo to a new date so when I run the query, it doesn't return empty.

Reply:
I could make validTo to be seven days from the current date.

Reply:
That sounds good!